tests/Browser/helpers/constants.js

```diff
@@ -21,7 +21,7 @@ export const MANAGER_FIRST_VISIT = `${HOMEPAGE}/manager/first-visit`;
 export const HR_FIRST_VISIT = `${HOMEPAGE}/hr/first-visit`;
 
 export const PROFILE_ABOUT = `${HOMEPAGE}/profile/about`;
-export const PROFILE_SKILLS = `${HOMEPAGE}/profile/skills`;
+export const PROFILE_SKILLS = `${HOMEPAGE}/profile/skills-old`;
```

Comment:
I'm not sure if this is meant to be part of this PR, but the Testcafe test for the profile has [not been updated](https://github.com/GCTC-NTGC/TalentCloud/blob/feature/profile-skills-finalized/tests/Browser/4-profile.testcafe.js#L18-L35) for the new skills page and so the test fails.

Reply:
I'll create some issues to update testcafe tests for the new Application and Profile